gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/FsJobStatusRetriever.java

```diff
@@ -83,7 +84,7 @@ public FsJobStatusRetriever(Config config, MultiContextIssueRepository issueRepo
       }
       return jobStatuses.iterator();
     } catch (IOException e) {
-      log.error("IOException encountered when retrieving job statuses for flow: {},{},{}", flowGroup, flowName, flowExecutionId, e);
+      log.error(String.format("IOException encountered when retrieving job statuses for flow: %s,%s,%s", flowGroup, flowName, flowExecutionId), e);
```

Comment:
I changed this because my understanding of signature, `(String, Object...)`, is that it treats all trailing args as format string interpolations, not recognizing that the final one might be `? extends Throwable`.  thus it won't print the stacktrace, as `(String, Throwable)` would.  if that's incorrect, or I misinterpreted the intent here, let's revert these lines.